pkg/relayer/proxy/synchronous.go

```diff
@@ -59,6 +58,7 @@ type synchronousRPCServer struct {
 // relay requests and forwards them to the supported proxied service endpoint.
 // It takes the serviceId, endpointUrl, and the main RelayerProxy as arguments
 // and returns a RelayServer that listens to incoming RelayRequests.
+// TODO_BLOCKER(red0ne): Use TCP server for the Gateway->RelayMiner communication.
```

Comment:
Ditto BLOCKER/IMPROVE.

I might even call it "research". 

I'm thinking about what will bring more adoption & customers and don't see issues with HTTP in the short term.